static/src/javascripts/projects/common/modules/commercial/gustyle/gustyle.js

```diff
@@ -33,12 +35,19 @@ define([
             };
 
         fastdom.write(function () {
-            this.$slot.addClass('gu-style');
+            var classList = 'gu-style' + ((this.isContentPage()) ? ' gu-style--unboxed' : '');
+
+            this.$slot.addClass(classList);
             this.$slot.prepend($.create(template(labelTpl, { data: merge(templateOptions) })));
 
             toggles = new Toggles(this.$slot[0]);
             toggles.init();
         }.bind(this));
     };
+
+    Gustyle.prototype.isContentPage = function () {
+        return !!config.page.isContent;
```

Comment:
Generally speaking I do think a viewmodel should act as a proxy for everything the view depends upon. So I can see advantages as well as disadvantages.